src/social-network-provider/facebook.com/UI/injectPostInspector.tsx

```diff
@@ -3,7 +3,7 @@ import { isMobileFacebook } from '../isMobile'
 import { PostInfo } from '../../../social-network/ui'
 import { injectPostInspectorDefault } from '../../../social-network/defaults/injectPostInspector'
 
-const defaultBehavior = injectPostInspectorDefault({
+export const defaultBehavior = injectPostInspectorDefault({
```

Comment:
Used where?

Reply:
![image](https://user-images.githubusercontent.com/15797507/63592223-a5b71d00-c5e3-11e9-9fcf-ef7fc84c074a.png)

Reply:
If it is only used in facebook support, you should name it as defaultBehaviorFacebook to avoid importing by mistakes. If it is used as a global default, don't do it.

Reply:
already set to nop on the twitter side.